assets/src/dashboard/app/views/shared/storyListView.js

```diff
@@ -76,49 +71,54 @@ const PreviewContainer = styled.div`
   display: inline-block;
   position: relative;
   overflow: hidden;
-  width: ${({ theme }) => theme.DEPRECATED_THEME.previewWidth.thumbnail}px;
-  height: ${({ theme }) =>
-    theme.DEPRECATED_THEME.previewWidth.thumbnail / FULLBLEED_RATIO}px;
+  width: 33px;
+  height: ${33 / FULLBLEED_RATIO}px;
   vertical-align: middle;
-  border-radius: ${({ theme }) =>
-    theme.DEPRECATED_THEME.storyPreview.borderRadius}px;
-  border: ${({ theme }) => theme.DEPRECATED_THEME.borders.gray75};
+  border-radius: ${({ theme }) => theme.borders.radius.small};
 `;
 
 const ArrowIcon = styled.div`
-  width: ${({ theme }) => theme.DEPRECATED_THEME.table.headerContentSize}px;
+  width: 32px;
   height: 100%;
   display: inline-grid;
-  color: ${({ theme }) => theme.DEPRECATED_THEME.colors.gray900};
+  color: ${({ theme }) => theme.colors.fg.primary};
   vertical-align: middle;
 
   svg {
     visibility: ${({ active }) => (active ? 'visible' : 'hidden')};
-    ${({ asc }) => asc && { transform: 'rotate(180deg)' }};
+    transition: transform 0.15s;
+
+    ${({ asc }) =>
+      asc &&
+      css`
+        transform: rotate(180deg);
+      `};
   }
 `;
 
+const EmptyIconSpace = styled.div`
+  height: 32px;
+  width: 32px;
+`;
+
 const ArrowIconWithTitle = styled(ArrowIcon)`
-  display: ${({ active }) => (active ? 'inline-grid' : 'none')};
-  margin-left: 6px;
-  margin-top: -2px;
+  display: ${({ active }) => !active && 'none'};
+  position: absolute;
+  top: 16px;
 
-  @media ${({ theme }) => theme.DEPRECATED_THEME.breakpoint.largeDisplayPhone} {
+  @media ${({ theme }) => theme.breakpoint.mobile} {
     margin-left: 4px;
   }
 `;
 
-const SelectableTitle = styled.span.attrs({ tabIndex: 0 })`
-  color: ${({ theme }) => theme.DEPRECATED_THEME.colors.bluePrimary};
-  font-weight: 500;
+const SelectableTitle = styled(Text).attrs({ tabIndex: 0 })`
+  color: ${({ theme }) => theme.colors.fg.linkNormal};
   cursor: pointer;
 `;
 
-const SelectableParagraph = styled(Paragraph2).attrs({
-  tabIndex: 0,
-  onFocus: onFocusSelectAll,
-  onBlur: onBlurDeselectAll,
-})``;
+const HeavyTitle = styled(Text)`
+  font-weight: 700;
```

Comment:
The font-weight of the `th` are `700` while the boldness of text is `500`. Didn't choose to use a `<Headline />` here because of the other headings are `spans`.

However, boldness of `<Text />` is `500` and the designs want `700`

Reply:
Good call, seems like enough of a one off.

Reply:
Since these are selectable, do you mind adding focus state for keyboard?
![Screen Shot 2021-02-25 at 2 27 11 PM](https://user-images.githubusercontent.com/10720454/109220009-9d5e1600-7775-11eb-9542-8bcb405ed759.png)